src/main/java/net/glowstone/generator/objects/IceSpike.java

```diff
@@ -25,6 +27,8 @@ public boolean generate(World world, Random random, int sourceX, int sourceY, in
         }
         boolean succeeded = false;
         int stemRadius = Math.max(0, Math.min(MAX_STEM_RADIUS, tipRadius - 1));
+        Set<Material> materialSet = new HashSet<>(Arrays.asList(MATERIALS));
```

Comment:
Would much rather have the MATERIALS variable be directly converted to a HashSet.

Reply:
Yes. I agree with you.

Reply:
EnumSet instead of HashSet?